webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/EventListDisplayer.scala

```diff
@@ -127,11 +131,13 @@ class EventListDisplayer(
       displayEvents(for {
         parsed   <- tryo(parse(jsonInterval)) ?~! s"Error when trying to parse '${jsonInterval}' as a JSON datastructure with fields 'start' and 'end'"
         startStr <- parsed \ "start" match {
-                      case JString(startStr) => tryo(new Timestamp(DateTime.parse(startStr, format).getMillis)) ?~! s"Bad format for start date, was execpting '${format.toString}' and got '${startStr}'"
+                      case JString(startStr) if startStr.isEmpty => tryo(new Timestamp(0))
```

Comment:
`Timestamp(0)` (or for the current system millis) can throw an exception? If not, there is no use for the `tryo`, you can just use `Full(Timestamp(0))`